Parse/ParseInstallation.cs

```diff
@@ -220,13 +220,13 @@ public ParseInstallation()
     internal override Task SaveAsync(Task toAwait, CancellationToken cancellationToken) {
       Task platformHookTask = null;
       if (CurrentInstallationController.IsCurrent(this)) {
-        SetIfDifferent("deviceType", ParseClient.PlatformHooks.DeviceType);
-        SetIfDifferent("timeZone", ParseClient.PlatformHooks.DeviceTimeZone);
-        SetIfDifferent("localeIdentifier", GetLocaleIdentifier());
-        SetIfDifferent("parseVersion", GetParseVersion().ToString());
-        SetIfDifferent("appVersion", ParseClient.PlatformHooks.AppBuildVersion);
-        SetIfDifferent("appIdentifier", ParseClient.PlatformHooks.AppIdentifier);
-        SetIfDifferent("appName", ParseClient.PlatformHooks.AppName);
+        ForceSetIfDifferent("deviceType", ParseClient.PlatformHooks.DeviceType);
```

Comment:
I think the correct solution for this would be to refactor `SetIfDifferent` so that it also forces `Remove`, and not make another `ForceSetIfDifferent`. `SetIfDifferent` by definition should force update.